octoprint_octorelay/driver.py

```diff
@@ -26,10 +25,14 @@ def open(self):
 
     def is_closed(self) -> bool:
         """Returns the logical state of the relay."""
-        GPIO.setwarnings(False)
-        GPIO.setup(self.pin, GPIO.OUT)
-        pin_state = bool(GPIO.input(self.pin))
-        GPIO.setwarnings(True)
+        request = request_lines(
```

Comment:
DNRY: this can probably be reused by keeping the request as a single prop of Relay

Reply:
addressed